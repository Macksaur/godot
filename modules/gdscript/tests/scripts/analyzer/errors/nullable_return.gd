func test():
    print(foo().begins_with("")) # Should error


func foo() -> String?:
    return null