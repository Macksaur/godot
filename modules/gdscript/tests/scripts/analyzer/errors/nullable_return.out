GDTEST_ANALYZER_ERROR
Potentially unsafe for the nullable return of "foo" to call "begins_with()".
